sound/soc/sof/topology.c

```diff
@@ -2633,19 +2633,31 @@ static void sof_dai_set_format(struct snd_soc_tplg_hw_config *hw_config,
 	}
 }
 
-/* set config for all DAI's with name matching the link name */
+/* Send IPC and set config for all DAI's with name matching the link name */
```

Comment:
@bardliao can we please expand this comment to make it explicit that we only support common config for all DAIs matching the DAI link for now.

Reply:
> @bardliao can we please expand this comment to make it explicit that we only support common config for all DAIs matching the DAI link for now.

Done